lib/freeathome.js

```diff
@@ -72,14 +72,22 @@ module.exports = class FreeAtHomeApi {
         this.adapter.setObjectNotExists(serialNumber, {
             type: 'device',
             common: {
-                name: device.typeName
+                name: device.typeName,
+                read: true,
+                write: false,
```

Comment:
If I understand the API correctly, ioBroker expects a type property in this object. Can we add that here?

Reply:
I think the type "string"